### DISCUSSION TITLE
Database error after starting the world server after a new installation

### DISCUSSION THREAD

**Discussion by TenebrisWoW** [azerothcore/azerothcore-wotlk#10070] [General]
Hi, everyone!

I created a server myself for the first time. After the world server was started for the first time, the databases were inserted. The following error occurs when starting:

`In mysql_stmt_prepare() id: 54, sql: "SELECT entry, difficulty_entry_1, difficulty_entry_2, difficulty_entry_3, KillCredit1, KillCredit2, modelid1, modelid2, modelid3, modelid4, name, subname, IconName, gossip_menu_id, minlevel, maxlevel, exp, faction, npcflag, speed_walk, speed_run, detection_range, scale, `rank`, dmgschool, DamageModifier, BaseAttackTime, RangeAttackTime, BaseVariance, RangeVariance, unit_class, unit_flags, unit_flags2, dynamicflags, family, trainer_type, trainer_spell, trainer_class, trainer_race, type, type_flags, lootid, pickpocketloot, skinloot, PetSpellDataId, VehicleId, mingold, maxgold, AIName, MovementType, ctm.Ground, ctm.Swim, ctm.Flight, ctm.Rooted, ctm.Chase, ctm.Random, ctm.InteractionPauseTimer, HoverHeight, HealthModifier, ManaModifier, ArmorModifier, ExperienceModifier, RacialLeader, movementId, RegenHealth, mechanic_immune_mask, spell_school_immune_mask, flags_extra, ScriptName FROM creature_template ct LEFT JOIN creature_template_movement ctm ON ct.entry = ctm.CreatureId WHERE entry = ?"
Table 'acore_world.creature_template_movement' doesn't exist`

Can somebody help me with it?

**Comment by TenebrisWoW** [comment]
I would be really happy if someone would help me as a newbie. Maybe someone can send me the part of the database. I've searched the internet and can't find a solution. I've already invested many hours and don't want to give up.

**Comment by Nyeriah** [comment]
Hello,

It seems you're missing a SQL update. Enable the database updater in your config files and try starting the server again. Let me know how if it works out for you.

Regards.

**Comment by TenebrisWoW** [comment]
Thanks for the answer! I will do it immediately. But it probably won't work. One moment!

**Comment by TenebrisWoW** [comment]
Updating Auth database...
>> Auth database is up-to-date! Containing 2 new and 26 archived updates.
 
Updating Character database...
>> Character database is up-to-date! Containing 4 new and 40 archived updates.
 
Updating World database...
>> Applying update "2021_12_29_01.sql" '4170551'...
**mysql: [Warning] Using a password on the command line interface can be insecure.♪◙ERROR 2006 (HY000) at line 4: MySQL server has gone away♪◙**
Applying of file 'D:/TenebrisCore/data/sql/updates/db_world/2021_12_29_01.sql' to database 'acore_world' failed! If you are a user, please pull the latest revision from the repository. Also make sure you have not applied any of the databases with your sql client. You cannot use auto-update system and import sql files from AzerothCore repository with your sql client. If you are a developer, please fix your sql query.
Could not update the World database, see log for details.
Closing down DatabasePool 'acore_world'.
Asynchronous connections on DatabasePool 'acore_world' terminated. Proceeding with synchronous connections.
All connections on DatabasePool 'acore_world' closed.
Closing down DatabasePool 'acore_characters'.
Asynchronous connections on DatabasePool 'acore_characters' terminated. Proceeding with synchronous connections.
All connections on DatabasePool 'acore_characters' closed.
Closing down DatabasePool 'acore_auth'.
Asynchronous connections on DatabasePool 'acore_auth' terminated. Proceeding with synchronous connections.
All connections on DatabasePool 'acore_auth' closed.



The update that doesn't work doesn't include the creature_template_movement!

**Reply by TenebrisWoW** [reply]
If I delete the tables from the world database and only the updates until December 28th. 2021, then he brings the following error at the end:

**Closing down DatabasePool 'acore_world'.
Asynchronous connections on DatabasePool 'acore_world' terminated. Proceeding with synchronous connections.
All connections on DatabasePool 'acore_world' closed.
Could not prepare statements of the World database, see log for details.**

**Comment by TenebrisWoW** [comment]
I would like to take this opportunity to thank Skjalf / Nyeriah very much! Then, with their help, the bug was ultimately fixed. Here is the corresponding passage from the Wiki as a future solution for seekers:

ACE00023 My Worldserver closes when autoupdater, I get:

**ERROR 2013 (HY000) at line 4: Lost connection to MySQL server during query
This is most likely due to your MySQL server's max_allowed_packet setting is too low. See this or run the command SET GLOBAL max_allowed_packet=1073741824; in your SQL client (HeidiSQL, SQLyog, etc.) to update your max_allowed_packet.**

**This value will reset the next time your SQL server restarts** and it may be necessary to run this query again in the future.

**Comment by MCVital** [comment]
Loading LFG rewards...
>> Loaded 0 lfg dungeon rewards. DB table `lfg_dungeon_rewards` is empty!
